src/styles/utilities/whiteSpace/index.js

```diff
@@ -5,4 +5,7 @@ export default {
     preWrap: {
         whiteSpace: 'pre-wrap',
     },
+    pre: {
+        whiteSpace: 'pre',
+    },
```

Comment:
Is this really necessary when this prop is supported on all platforms?

Reply:
> Is this really necessary when this prop is supported on all platforms?

No, we could add it directly to baseFontStyle, I just thought it would be more consistent since we're already have noWrap and preWrap.
Should I move it to baseFontStyle directly?

Reply:
> Should I move it to baseFontStyle directly?

yeah, Have you checked that `whitespace: pre` is supported on Android and iOS. I guess not.